gson/src/main/java/com/google/gson/internal/Primitives.java

```diff
@@ -60,7 +60,7 @@ public static boolean isWrapperType(Type type) {
    *     wrap(String.class) == String.class
    * </pre>
    */
-  @SuppressWarnings("unchecked")
+  @SuppressWarnings({"unchecked", "MissingBraces"})
```

Comment:
To be honest I haven't heard of `MethodType` before. Using it here feels a bit like API misuse to me, though unfortunately `java.lang.Class` itself does not provide methods for wrapping and unwrapped.

Though according to https://developer.android.com/reference/java/lang/invoke/MethodType `MethodType` only exists on Android since API level 26, and we currently specify API level 21 as minimum. So using `MethodType` is not an option then anyways.

Would it be fine to keep the suppression then?

Reply:
OK.